packages/terra-toolkit-docs/.npmignore

```diff
@@ -0,0 +1,8 @@
+*.log
```

Comment:
Thoughts on migrating to the files package.json key for defining what should be released? It seems we sometimes end up with more in the release than needed with this pattern

Reply:
I think generally it's safer to publish more than publish a version where someone forgot to include a file? 🤷‍♂️